setup_emul_build_env.sh

```diff
@@ -19,7 +19,7 @@ export OMP_NUM_THREADS=1
 
 # Use gold if it's available for faster linking.
 if which gold >/dev/null 2>&1 ; then
-    export CFLAGS='-fuse-ld=gold'
+    export CFLAGS='-fuse-ld=ld'
```

Comment:
Seeing something wrong with `-fuse-ld=gold`?

Reply:
emmm. not sure we should merge in this change?

Reply:
Yes, as I mentioned on Slack to @yuyihe, this manual change is no longer necessary (since I added the `if`).